brainiak/reconstruct/iem.py

```diff
@@ -0,0 +1,290 @@
+#  Copyright 2018 David Huberdeau & Peter Kok
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+"""Inverted Encoding Model (IEM)
+
+    Method to decode and reconstruct features from data.
+
+    The implementation is roughly based on the following publications:
```

Comment:
I think it would be useful to briefly comment on what's the difference between this implementation and what's proposed in the paper.

Reply:
Yes, I added a description.